### ISSUE TITLE
BB#54 Kornvin 3. 

### ISSUE THREAD

**Issue by Bbbolaget-karl** [sweco-seukni/Bbbolaget#36]
Bryggs på kakan av #52
Idémässtigt, för att få upp OG tillräcklgt, lutas det i nuläget åt att mäska två gånger. 
först 5,5 kg Maris Otter. Detta kokas på spisen tills det känns lagom att tillsätta den till braumeistern
Därefter ytterligare 5,5 kg mäsk. 
ska vi ha nån procent av kristallmalt?

**Comment by sweco-seukni** [comment]
Jag tycker kristall och kanske nån procent Special B också, om vi har det hemma, inte för mycket men lite komplexare än den första tycker jag hade varit bra, 150g av varje?

**Comment by Bbbolaget-karl** [comment]
ok, 
Maris Otter pale Ale 98% 10 780 g 
crystal 100, 1%, 110 g
Special B, 1%, 110 g

**Comment by Bbbolaget-karl** [comment]
Mäsktemp? 
version två på 66°C
Version ett på 66°C enl recept men verkar varit 68°C vid t=slut

Alltså, jag förespråkar 68°C. 

För övrigt har BW-boken också kommit. Kan se om den fingervisar i nån riktning.

**Comment by Bbbolaget-karl** [comment]
mäsktemp 67 gr
og på första.. 1060.
 kok på första kl 1050

**Comment by Bbbolaget-karl** [comment]
31 g @ 11,7 Alfa ger 40 ibu. target
60 g @ 6 Alfa whitbread goldings variaty. 40 ibu till

**Comment by Bbbolaget-karl** [comment]
händelseförlopp
efter mäskning 1, utan lakning, överfördes denna mäsk till gamla grytan och kokades där varpå nästa maltomgång förbereddes till BM. All specialmalt gick i mäsk 2. 
Innan mäsk 2 var färdig hade mäsk 1 kokat i säkert 2 och 1/2 timme, inkluderat ett stopp i koket då plattan slog av sig själv. 
Dessa två kokades parallellt. När det var ungefär 12 liter kvar i grytan överfördes mäsk från BM till denna tills en ungefärlig volym av 24 liter uppnåddes. Därefter kokades båda kärlen parallellt ytterligare en stund (30 min) varpå ytterligare lite överfördes från BM. 
Nytt SG togs från grytan och uppmättes till ungefär 1.087. Koktiden sattes därefter till 90 min. 

ur BM tappades 6 liter ut som sedan slängdes. Denna vört hade OG ca 1065 (varm)

**Comment by Bbbolaget-karl** [comment]
jäser på bra. 
efter 1 dygn nere på 1065
efter ytterligare 6 h, på 1050. 
2 dygn efter, bubblar i jäsröret. 

jästen har värmt vörten, varit uppe på 26 °C som mest. 

Har haft problem med att locket inte slutit tätt, nu åtgärdat (för tillfället?)

**Comment by Bbbolaget-karl** [comment]
21 grader idag. jäsning verkar avstannat. försökte rusa (rouse) Jästen genom att skaka lite på behållaren. 


lite rädd för att jäsningen stack iväg i temp såpass mycket. skulle man bara haft halva kakan? lägre omgivande temp?

**Comment by sweco-seukni** [comment]
Brukar du kolla temperaturen under jäsning? Hur mycket skiljer det? Har du kollat FG, den kanske intebehöver jäsa mer

**Comment by Bbbolaget-karl** [comment]
Ja, ibland. Det är alltså en sån där LCD-temp som är klistrad utanpå jäskärlet, inget invändigt. 
Kollade FG nu. 1032. Den ska nå 80% utjäsningsgrad, wlp007. vi är på ungefär 68-70% nu. Men den jäser fortfarande. Om än inte så mycket. Ska skaka om ikväll igen.

**Comment by sweco-seukni** [comment]
Låter som jäaningen går ok ändå va, kan väl vara att wlp007 presterar något sämre när alkoholen blir högre, skaka lite och kolla fg i helgen igen

**Comment by Bbbolaget-karl** [comment]
Har skakat. Kollar FG på söndag. Däremellan Düsseldorf. 
Telefonen pajade idag så jag är utanför täckningen. 

Man kan normalt tänka sig lägre utjäsningsgrad när det är hög alkohålhalt. Så det har du rätt i. Är uppe på 10% abv redan, hursomhelst. 

för att nå 80% ska vi ner till 1020. Det är inte dit vi vill. Snarare 1025-28. Vi får se, helt enkelt. =)

**Comment by Bbbolaget-karl** [comment]
SG idag16 september. 1030

**Comment by Bbbolaget-karl** [comment]
söt, lakrits, alkohålig. Svagt bittrad?

**Comment by Bbbolaget-karl** [comment]
Fg 1034¡ behövs kalibrering?

**Comment by Bbbolaget-karl** [comment]
wlp007
